src/components/Form.tsx

```diff
@@ -50,6 +53,11 @@ export const Form = ({ showForm, setShowForm }: Props): JSX.Element => {
       if (selectedOption) {
         console.log('saved'); // TODO Implement save callback
         closeForm();
+        setShowSubButtons(false);
+        setShowAlert(true);
+        setTimeout(() => {
+          setShowAlert(false);
+        }, 3000);
```

Comment:
how about we add the timeout in AlertContext, so when any component calls this, it will autoclose it. make the timeout as configurable is also good. with this, we then can export the `showAlert()` function like:
```
const showAlert = () => {
  setShowAlert(true)
  setTimeout(() => {
    setShowAlert(false);
  }, 3000);
}

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/7/commits/5df22cda465ec63c43e212b168bef59b07497d02, thank you